src/components/Checkout/KivaCreditPayment.vue

```diff
@@ -1,18 +1,19 @@
 <template>
 	<kv-button
-		type="submit"
-		class="smaller"
+		value="submit"
```

Comment:
Assuming this is inside a form we probably want it to remain `type="submit"` instead of value

Reply:
Hmmm, yea, maybe I need to add the `prevent` modifier back to the click b/c we don't actually submit the form unless JS isn't loaded...